### PR TITLE
Weather App

### PR DESCRIPTION
**Pull request by LoboArkano**
In this JavaScript project I put in practice my knowledge about async/await and APIs.

***Project Requirements***

- [X] Set up a blank HTML document with the appropriate links to your JavaScript and CSS files.
- [X] Write the functions that hit the API. You’re going to want functions that can take a location and return the weather data for that location. For now, just console.log() the information.
- [X] Write the functions that process the JSON data you’re getting from the API and return an object with only the data you require for your app.
- [X] Set up a simple form that will let users input their location and will fetch the weather info (still just console.log() it).
- [X] Display the information on your webpage!
- [X] Add any styling you like!
- [X] You should be able to search for a specific location and toggle displaying the data in Fahrenheit or Celsius.
 
***General Requirements***

- [x] Add a descriptive README file to your repository
- [x] The README must include instructions on how to use the project on your machine
- [x] Files not related to the project should not be commited and added to .gitignore.
- [x] Use descriptive commit messages
- [x] Do not commit debugging instructions

***GitFlow***

- [x] Create a development branch and set it as a default branch in your repository.
- [x] Create a feature-branch branch (give it a descriptive name)

### CODE REVIEW

**Comment by LoboArkano** [comment]
By mistake, I merged the branch before the code review. This is the original pull request: [pull request](https://github.com/LoboArkano/Weather-App/pull/1)

**Comment by LoboArkano** [comment]
By mistake, I merged the branch before the code review. This is the original pull request: [pull request](https://github.com/LoboArkano/Weather-App/pull/1)

File: package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "css-loader": "^4.3.0",
+    "style-loader": "^1.2.1",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^3.3.12"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch"
+  },
```
Comment (addod19) [inline package.json]: Kindly change these to
 ```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @LoboArkano, congratulations on a great job done :fireworks: Awesome!!! However, before we can approve the project, we need to address the issue below: :+1: 
- `Package.json` file, let us edit it to look like the required scripts and also updated the README file accordingly

Once this is fixed, kindly submit another code review :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by stratospherique — APPROVED** [review]
### Great work implementing the suggested changes :clap: Congrats, the project is approved :heavy_check_mark: You may merge :+1: :+1: 

Keep up the good work :muscle: 
**_Ahmed Mahfoudh_**

**Pull request merged**